config.example.js

```diff
@@ -31,6 +29,8 @@ module.exports = {
   // The URI containing JSON metadata about your contract
   // Please follow: https://docs.opensea.io/docs/contract-level-metadata
   contractURI: "ipfs://...",
+  // The creator's STARS address
+  creator: "stars1...",
```

Comment:
I think its better to call this `account` or `address` since its most likely the creator that is running the script.

Reply:
Done.